src/web/Dockerfile

```diff
@@ -7,7 +7,8 @@ ENV NODE_ENV=production
 ENV SESSION_SECRET=${SESSION_SECRET:-whateversession}
 ENV CASSANDRA_CLUSTER_IPS=${CASSANDRA_CLUSTER_IPS:-192.168.1.10}
 ENV CASSANDRA_CLUSTER_PORT=${CASSANDRA_CLUSTER_PORT:-9043}
-ENV CASSANDRA_FLASK_SERVER_URL=${CASSANDRA_FLASK_SERVER_URL:-http://192.168.1.10:5001}
+ENV CASSANDRA_FLASK_SERVICE_URL=${CASSANDRA_FLASK_SERVER_URL:-http://192.168.1.10:5001}
```

Comment:
How come the default URL for this and the one in src/web/server/app/utils/settings.ts is different?

Reply:
Opps. I'll take note of that in the next pull request. (I felt like server isn't the right word for that service.)